seeds.js

```diff
@@ -15,28 +15,37 @@ const seedMembers = [
         firstName: "Xavi",
         lastName: "Hernandez",
         studentID: "123456789",
+        degree: "Doktoraa",
```

Comment:
Typo.

Reply:
Ah, I forgot to reverse it after testing `enum`.